src/linear_algebra/projective_space/subspace.lean

```diff
@@ -127,6 +121,30 @@ instance : complete_lattice (subspace K V) :=
 instance subspace_inhabited : inhabited (subspace K V) :=
 { default := ⊤ }
 
+/-- The span of a set of points is contained in a subspace iff the set of points is contained in the
+subspace. -/
+lemma span_le_subspace_iff {S : set (ℙ K V)} {W : subspace K V} : span S ≤ W ↔ S ⊆ W :=
+by { exact gi.gc S W }
+
+/-- If a set of points is a subset of another set of points, then its span will be contained in the
+span of that set. -/
+lemma span_mono {S T : set (ℙ K V)} (h : S ⊆ T) : span S ≤ span T :=
+begin
```

Comment:
Isn't this a 1-line consequence of the Galois insertion as well? Should be `galois_connection.monotone_l`.

Reply:
Thanks for implementing Johan's suggestion.

It's really helpful for reviewing if you can click "Resolve conversation" after you've addressed a comment like this.